data/redirects.toml

```diff
@@ -1,3 +1,15 @@
+[[redirects]]
+from   = "/newsletters/howwecare"
+to     = "/newsletters"
+status = 301
+force  = true
+
+[[redirects]]
+from   = "/newsletters/howwecare/"
+to     = "/newsletters"
+status = 301
+force  = true
```

Comment:
I don’t think you need both. It should get the slash and no slash versions with one redirect.